airbyte-cdk/bulk/core/load/build.gradle

```diff
@@ -10,3 +10,29 @@ dependencies {
     testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.8.1")
     implementation "org.jetbrains.kotlin:kotlin-reflect:2.0.20"
 }
+
+sourceSets {
+    integrationTest {
+    }
```

Comment:
Do these require paths?

Reply:
I guess it defaults?

Reply:
Is the point to support this: ```Add a new integrationTest task+classpath, to avoid accidentally breaking any micronaut stuff in unit tests. Also updates our github actions (I think?) to call that task as needed.```

Like the point is to prevent connector integration tests from pulling this in? Maybe a clearer/pattern-following name like `test-integration-cdk`?

Reply:
welp, my comment on this got hidden https://github.com/airbytehq/airbyte/pull/45634#discussion_r1775637459 (tl;dr yes, it populates the paths by default. I should probably just make that an in-code comment 🤔 )

it's more that I'm declaring Singletons without `@Requires` (b/c I didn't want to add more plumbing into CliRunner/AirbyteConnectorRunner). So putting this stuff into a separate task+classpath (instead of just `test`) hopefully avoids annoying duplicate bean errors in the cdk unit tests. (connector integration tests wouldn't be affected either way, since they only depend on testFixtures)

so I think integrationTest is the right name? (but also, I don't really feel strongly :P )
* test -> cdk unit tests
* testFixtures -> cdk tooling for connector tests
* integrationTest -> cdk integration tests